packages/core/src/root_files/localstorage-sync.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Deriv</title>
+        <meta charset='utf-8' />
+        <meta name='referrer' content='origin' />
+        <script>
+            window.addEventListener('message', function(message) {
+                var restricted_origins = [
+                    'https://smarttrader.deriv.app', 
+                    'https://smarttrader-staging.deriv.app'
```

Comment:
Wouldn't these domains have to be deriv.app? (IIUC) As this file will get imported into SmartTrader, and Deriv.app will import SmartTrader's implementation.

Reply:
You're right in that this file will get imported in SmartTrader, however we're still operating within the scope of Deriv.app inside the iframe, the `message.origin` should reflect SmartTrader's origin. Please correct me if I'm wrong.

Reply:
You are correct. Thank you. 👍